include/zephyr/settings/settings.h

```diff
@@ -136,6 +139,9 @@ struct settings_handler_static {
 	const char *name;
 	/**< Name of subtree. */
 
+	int cprio;
+	/**< Priority of commit, lower value is higher priority */
+
```

Comment:
Should these additions be guarded by a Kconfig, and perhaps made experimental?

Reply:
I don't think this is required.